.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+    "trailingComma": "none",
+    "tabWidth": 4,
```

Comment:
Seeing some files that are using 2-space indentation still while others are using 4-space indentation.